model.py

```diff
@@ -5836,17 +5839,6 @@ def as_delivery_mechanism_for(self, licensepool):
             if lpdm.resource == self:
                 return lpdm
 
-    def set_mirrored_elsewhere(self, media_type):
-        """We don't need our own copy of this resource's representation--
-        a copy of it has been mirrored already.
-        """
-        _db = Session.object_session(self)
-        if not self.representation:
-            self.representation, is_new = get_one_or_create(
-                _db, Representation, url=self.url, media_type=media_type)
-        self.representation.mirror_url = self.url
-        self.representation.set_as_mirrored()
-
```

Comment:
We no longer care whether a resource was 'mirrored' by someone else -- either we mirrored it ourselves or we're using someone else's URL.